README.md

```diff
@@ -83,9 +85,11 @@ You will also need to set `OIDC_URL` and `OIDC_CLIENT_NAME` with the correct val
 
 The MLSpace application is a standard CDK application and can be deployed just as any CDK application is deployed:
 ```
+# From / root directory
```

Comment:
I think this should say "project root" or something because `/` and `root` invoke the root of the file system.